rules/code-quality/tests/Rector/Class_/CompleteDynamicPropertiesRector/CompleteDynamicPropertiesRectorTest.php

```diff
@@ -30,7 +30,7 @@ protected function getRectorClass(): string
         return CompleteDynamicPropertiesRector::class;
     }
 
-    protected function getPhpVersion(): string
+    protected function getPhpVersion(): int
     {
         return PhpVersionFeature::BEFORE_UNION_TYPES;
```

Comment:
The `BEFORE_` constants now can be removed.

Their calls should be replaced with:

```suggestion
        return PhpVersionFeature::UNION_TYPES - 1;
```

Reply:
implemented.